crates/edr_napi/src/solidity_tests/config.rs

```diff
@@ -413,6 +417,20 @@ impl TryFrom<FuzzConfigArgs> for FuzzConfig {
     }
 }
 
+impl SolidityTestRunnerConfigArgs {
+    pub fn try_get_test_filter(&self) -> napi::Result<TestFilterConfig> {
```

Comment:
```suggestion
    pub fn try_to_get_test_filter(&self) -> napi::Result<TestFilterConfig> {
```

Nit: I have a preference for readable function names, albeit slightly longer.

Feel free to decide either way.